projects/vdk-plugins/vdk-impala/README.md

```diff
@@ -26,6 +26,28 @@ For example
         job_input.execute_query("select 'Hi Impala!'")
 ```
 
+Most of the processing templates support quality checks that are used for preventing bad data going into production tables.
+The checks represent a callback function that is passed as an optional parameter to the job_input.execute_template() method.
```

Comment:
I don't get this. How come some templates support this and others don't? It seems pretty generic to me, so shouldn't all templates support it?

Reply:
Currently quality checks handling is implemented only for scd1, snapshot and insert templates so users won't be able to use them for scd2 and versioned templates until they are implemented for them as well.